packages/devtools/lib/src/ui/dragscroll.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:html';
+
+class DragScroll {
+  var mousemove = 'mousemove';
+  var mouseup = 'mouseup';
+  var mousedown = 'mousedown';
+  var EventListener = 'EventListener';
+
+// ??? what are these ???
+//  var addEventListener = 'add'+EventListener;
+//  var removeEventListener = 'remove'+EventListener;
+
+
+
+  num newScrollX;
+  num newScrollY;
+
+  var dragged = [];
+
+  void reset(int i, /*type?*/ eventListener) {
+    while( i < dragged.length) {
+      eventListener = dragged[i++];
+      eventListener = eventListener.container ?? eventListener;
+      /* what are mu mm md? mouse up, mouse move, and mouse down? */
+      eventListener.removeEventListener(mousedown, eventListener.md, false);
+      window.removeEventListener(mouseup, eventListener.mu, false);
+      window.removeEventListener(mousemove, eventListener.mm, false);
+    }
+
+    dragged = document.getElementsByClassName('dragscroll');
+
+    i = 0;
+    while( i < dragged.length) {
+      // is this (function) block a method? is function its name or a keyword?
+      (function(
```

Comment:
this was just a function that was defined and then called immediately. It is because JS has some weird scoping rules for var they were working around. just rip out that function and write the code inline.